src/main/java/frc/robot/hardware/phoenix6/Phoenix6Utils.java

```diff
@@ -1,23 +1,23 @@
-package frc.utils.ctre;
+package frc.robot.hardware.phoenix6;
 
 import com.ctre.phoenix6.StatusCode;
 import com.ctre.phoenix6.StatusSignal;
 
 import java.util.function.Supplier;
 
-public class PhoenixProUtils {
+public class Phoenix6Utils {
 
 	public static <T> StatusSignal<T> getRefreshedSignal(boolean refresh, StatusSignal<T> signal) {
```

Comment:
do with generic with ? like in the rest of the code

Reply:
can't, java won't let you